saltgui/static/scripts/api.js

```diff
@@ -133,13 +141,33 @@ class API {
     var functionToRun = args[0];
     args.shift();
 
-    var params = {
-      tgt: target,
-      fun: functionToRun,
-      client: "local"
-    };
-
-    if(args.length !== 0) params.arg = args.join(" ");
+    var params = {}
+
+    if(functionToRun == "salt.wheel.key.accept") {
+      // See https://docs.saltstack.com/en/latest/ref/wheel/all/salt.wheel.key.html#salt.wheel.key.accept
+      params.client = "wheel";
+      params.fun = functionToRun.substring(11);
+      params.match = target;
+      params.include_denied = true;
+      params.include_rejected = true;
+    } else if(functionToRun == "salt.wheel.key.reject") {
+      // See https://docs.saltstack.com/en/latest/ref/wheel/all/salt.wheel.key.html#salt.wheel.key.reject
+      params.client = "wheel";
+      params.fun = functionToRun.substring(11);
+      params.match = target;
+      params.include_accepted = true;
+      params.include_denied = true;
+    } else if(functionToRun == "salt.wheel.key.delete") {
+      // See https://docs.saltstack.com/en/latest/ref/wheel/all/salt.wheel.key.html#salt.wheel.key.delete
+      params.client = "wheel";
+      params.fun = functionToRun.substring(11);
+      params.match = target;
+    } else {
+      params.client = "local";
+      params.fun = functionToRun;
+      params.tgt = target;
+      if(args.length !== 0) params.arg = args.join(" ");
+    }
```

Comment:
I see a lot of code duplication here. I think this can be done a lot cleaner

Reply:
Agreed, working on it...

Reply:
Separated the client/fun/match handling from the rest, since these were present 3 times.

Reply:
done